includes/Traits/Stories_Script_Data.php

```diff
@@ -95,19 +57,10 @@ private function data() {
 
 		$field_states = fields_states();
 
-		$data = [
-			'orderlist' => $order_list,
-			'tag'       => Stories_Shortcode::SHORTCODE_NAME,
-			'views'     => $view_types,
-			'fields'    => $field_states,
+		return [
+			'views'  => $view_types,
+			'fields' => $field_states,
 		];
-
-		/**
-		 * Filter the script data.
-		 *
-		 * @param array $data Script data.
-		 */
-		return apply_filters( 'web_stories_script_data', $data );
```

Comment:
I don't think that this filter has now been removed. This was added by request in other code review.

Reply:
I really don't see the need for a filter here for such a low level util